cmd/acceptance-sentinel/main.go

```diff
@@ -2,26 +2,28 @@ package main
 
 import (
 	"context"
+	"crypto"
 	"fmt"
 	"time"
 
 	secretmanager "cloud.google.com/go/secretmanager/apiv1"
 	"github.com/ipfs/go-cid"
 	shell "github.com/ipfs/go-ipfs-api"
 	"github.com/kelseyhightower/envconfig"
-	"github.com/libp2p/go-libp2p-core/crypto"
 	"github.com/rs/zerolog/log"
 	secretmanagerpb "google.golang.org/genproto/googleapis/cloud/secretmanager/v1"
 
 	"github.com/areknoster/public-distributed-commit-log/cmd/acceptance-sentinel/internal/validator"
+	pdclcrypto "github.com/areknoster/public-distributed-commit-log/crypto"
 	"github.com/areknoster/public-distributed-commit-log/grpc"
 	"github.com/areknoster/public-distributed-commit-log/ipns"
 	"github.com/areknoster/public-distributed-commit-log/ratelimiting"
 	commiter "github.com/areknoster/public-distributed-commit-log/sentinel/commiter"
 	"github.com/areknoster/public-distributed-commit-log/sentinel/pinner"
 	"github.com/areknoster/public-distributed-commit-log/sentinel/sentinelpb"
 	"github.com/areknoster/public-distributed-commit-log/sentinel/service"
-	daemonstorage "github.com/areknoster/public-distributed-commit-log/storage/ipfs/daemon"
+	daemonstorage "github.com/areknoster/public-distributed-commit-log/storage/message/ipfs"
```

Comment:
```suggestion
	ipfsstorage "github.com/areknoster/public-distributed-commit-log/storage/message/ipfs"
```
In other places as well